fe/src/components/Users/Users.re

```diff
@@ -28,25 +34,59 @@ let prepareTeamIcons = teamName => {
   <span className=icon />;
 };
 
-let tableRow = (userForDisp: userForDisp) =>
-  <tr key={string_of_int(userForDisp.userNid)}>
-    <td className="posTd"> {string(string_of_int(userForDisp.pos))} </td>
-    <td className="codeTd"> {string(userForDisp.code)} </td>
-    <td className="nameTd"> {string(userForDisp.name)} </td>
-    <td className="teamTd"> {userForDisp.team |> prepareTeamIcons} </td>
+let getTrend = lastSummaryRatingDiff =>
```

Comment:
Tutaj nie dałoby rady zrobić porównań switchem?

Reply:
@ktrzos Jak to robiłem pamiętam że próbowałem na switchu i coś nie pykało i się wkurzyłem i zrobiłem na ifach.
Teraz siadłem znowu i zrobiłem na switchu. Switch domyślenie umie tylko porównywać (ewentualnie robić jednocześnie destrukcję i ją porównywać). Aby robić sprawdzanie jakiś logicznych warunków (jak tutaj sprawdzanie przedziałów) trzeba używać słówka when.